### PR TITLE
Update 1.13

### PR DESCRIPTION
**Pull request by Jrice1317**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,19 +12,18 @@ source:
...
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
+  skip: True # [py<37]
 
 requirements:
   host:
-    - python >=2.7
+    - python
     - pip
-    - setuptools >=42
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:23]: `setuptools` without pinning is still needed, https://github.com/colour-science/flask-compress/blob/079b2152c9a40c5eaad4a28491f384ccae6ab6c5/pyproject.toml#L2

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,19 +12,18 @@ source:
...
-    - setuptools_scm >=3.4
+    - setuptools_scm
     - wheel
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:26]: `toml` is missing. It's inside brackets https://github.com/colour-science/flask-compress/blob/079b2152c9a40c5eaad4a28491f384ccae6ab6c5/pyproject.toml#L2. `setuptools_scm[toml]` means `setuptools_scm` **with** `toml`

Comment (skupr-anaconda) [reply recipe/meta.yaml:26]: `setuptools_scm[toml]` should be:
```
- setuptools_scm
- toml
```

**Review by skupr-anaconda — APPROVED** [review]

**Review by ryanskeith — APPROVED** [review]
LGTM

**Pull request merged**